packages/airnode-deployer/src/cli/commands.ts

```diff
@@ -69,7 +68,7 @@ export async function remove(airnodeAddressShort: string, stage: string, cloudPr
 }
 
 export async function removeWithReceipt(receiptFilename: string) {
-  const receipt = parseReceiptFile(receiptFilename);
+  const receipt = parseReceiptFile(receiptFilename, getNodeVersion());
```

Comment:
You're calling `getNodeVersion` inside `parseConfig` but passing the value when calling `parseReceiptFile`. I think it would be better to choose one approach and stick to it if possible.